run-test.sh

```diff
@@ -18,17 +18,15 @@ fi
 
 # Launch a container running systemd
 CONTAINER="$(
-    docker run -d --rm \
-               --cap-add SYS_ADMIN --tmpfs /tmp --tmpfs /run --tmpfs /run/lock -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
-               -v "$PWD:/tests" "$TEST_IMAGE"
+    podman run -dit --rm -v "$PWD:/tests" "$TEST_IMAGE"
```

Comment:
Why do we need `-it` for Podman but didn't for Docker?

Reply:
We don't, was left over from debugging, will remove